src/selectors/profile.js

```diff
@@ -139,6 +139,64 @@ export const getCounterSelectors = (index: CounterIndex): CounterSelectors => {
   return selectors;
 };
 
+const getThreadsForPIDs = createSelector(
+  getThreads,
+  (threads) => {
+    let pid2threads = {};
```

Comment:
This name is confusing to me, is it supposed to be `pidToThreads`?  It's also more common to the codebase to use the `Map` data structure, although this is functionally the same.

Reply:
Yeah, it's a mapping from PIDs to thread objects. Using "2" for "to" in symbol names is just a (possibly bad) habit I picked up from somewhere...